pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -473,6 +476,16 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 
 GraphicsSettingsWidget::~GraphicsSettingsWidget() = default;
 
+void GraphicsSettingsWidget::onTextureFilteringChange()
+{
+	m_ui.swTextureFiltering->setCurrentIndex(m_ui.textureFiltering->currentIndex());
```

Comment:
This will update settings twice (setting the current index on the other combo will fire the signals on it, just like the user selecting the option would).

Easiest way would be to use `QSignalBlocker sb(m_ui.swTextureFiltering);`.

Reply:
Thanks, I had a feeling you'd know better :P I'll change that in a bit